.appveyor.yml

```diff
@@ -50,6 +50,12 @@ for:
       cmd: scripts/install_windows.bat %WINDOWS_HPCKIT_URL% %WINDOWS_CPP_COMPONENTS%
     build_script:
       cmd: scripts/build_windows.bat c++
+    artifacts:
```

Comment:
```suggestion
    # Delete this if you don't want to save installer logs
    artifacts:
```

Reply:
I think this comment in needed in multiple places where artefacts are defined. I will add it centrally in a separate PR.